Sources/SpelledPitch/PitchSpeller/Wetherfield/SpellingInverter.swift

```diff
@@ -32,6 +32,13 @@ extension SpellingInverter {
     
     // MARK: - Initializers
     
+    init(spellings: [Pitch.Spelling], parsimonyPivot: Pitch.Spelling = .d) {
+        let dict = spellings.enumerated().reduce(into: [Int: Pitch.Spelling]()) {
```

Comment:
Nit: you can get away with `.reduce(into: [:]) {` here, but no biggie.

Reply:
I think I will do this and then do away with all the $0's and $1's for clarity

Reply:
Ah, so because of the later usage, the compiler complained about not knowing the type of dict, so do you think it is better to have `dict: [Int: Pitch.Spelling]` or the explicit `reduct(into: [Int: Pitch.Spelling]())`?

Reply:
For now I am going with the former